README.md

```diff
@@ -11,16 +11,14 @@ This module implements the Thin Shell Demons regularization proposed in
 > MIUA 2015  
 
 
-There are two implementations of this approach available:
-1. (TSHDv4) An implementation taking advantage of the point set to point set registration
-   capabilities of the ITK v4 registration framework.
-2. (TSHD) An implementation started by Qingyu Zhao and updated to fix some remaining task
-   and run with ITK 5
 
 > :warning: **This module requires to be compiled against an ITK version with**  
 > - [PointSetToPointSetMetricWithIndexv4](https://github.com/InsightSoftwareConsortium/ITK/pull/2385)   
 > 
-> If you are compiling against an older version  use the ones inlcuded in this repository marked with h/hxx-unused (remove the -unused).
+
+<p align="center">
```

Comment:
Do we need HTML formatting? Why not put this image into the repository, and reference it using normal markdown syntax?

Reply:
I needed it to change the size of image and to center it. I could not get such control using markdown syntax.